src/entity/posts/utils.ts

```diff
@@ -378,6 +385,8 @@ export const createExternalLink = async (
       sourceId: UNKNOWN_SOURCE,
       url,
       canonicalUrl: url,
+      title,
+      image,
       sentAnalyticsReport: true,
       private: true,
       origin: PostOrigin.Squad,
```

Comment:
Should we not do something with this visible as well? (Line below)

Reply:
This is the article with unknown source which should still always be `visible: false`.

Reply:
That's not really correct, they also become visible when the title is available :)
They stay private at all times is what you mean.

This case would cause the error of it showing the share post, but not being able to load the actual title.

Reply:
Oh, in that case, my understanding was wrong. Thanks for pointing it out, let me update it.